BenchtopDCServo.h

```
@@ -77,6 +77,9 @@ class BenchtopDCServo final : public NonStepperMotorDrive {
     short Kinesis_ResetRotationModes() override;
     short Kinesis_SetRotationModes(int mode, int direction) override;
 
+    short Kinesis_SetHomingParams(int, int, int, int) override { return 0; };
+    short Kinesis_SetLimitSwitchParams(int, int, int, int, int) override { return 0; };
+
```

Might it be better to return a non-zero value (such as -1) to indicate an error? I know they are not supposed to be called when not supported, but though it might be better not to hide the error if it happens by some mistake.

(This applies to all overrides of these two functions where the features are not supported, of course.)